ush/fv3gfs_driver_grid.sh

```diff
@@ -99,6 +102,7 @@ export home_dir=${home_dir:-"$PWD/../"}
 export script_dir=$home_dir/ush
 export exec_dir=${exec_dir:-"$home_dir/exec"}
 export topo=$home_dir/fix/fix_orog
+export topo_am=$home_dir/fix/fix_am
```

Comment:
Your input terrain datasets should really be in 'fix_orog' with the others, not in fix_am.  I will get with @KateFriedman-NOAA to move them.  Then you can get rid of the topo_am variable.

Reply:
Sounds good.